Misc/NEWS.d/next/Library/2020-11-23-23-42-08.bpo-42299.Fdn4Wf.rst

```diff
@@ -0,0 +1,2 @@
+Removed the ``formatter`` module, which was deprecated in 3.4.
```

Comment:
You may also add (here, but maybe also in the What's New entry) that the module had no test!